### DISCUSSION TITLE
Error when setup teleport cluster to connect with dynamodb

### DISCUSSION THREAD

**Discussion by sandunsameera** [gravitational/teleport#49615] [Q&A] [answered]
Im getting below error on my jounalctl logs when creating teleport service: auth server requires a valid license file to start, please set the correct license_file path under auth_service section in your teleport config or put the license into the default search location at /var/lib/teleport/license.pem

**Comment by zmb3** [comment, marked as answer]
This means you are running a Teleport Enterprise build without a license file. 

If you are a Teleport Enterprise customer, then you need to provide the path to your license file in your configuration as the error message suggests. 

If you are not an enterprise customer then you should not be running an enterprise build. You will either need to build from source, or run a Community Edition build (so long as you meet the licensing requirements of the community edition).
